### PR TITLE
feat(mqtt extension): add tip for proxy sub

### PR DESCRIPTION
**Pull request by Kinplemelon**

### CODE REVIEW

File: src/i18n/Extension.js [hunk 1]
```diff
@@ -309,4 +309,8 @@ export default {
     en: 'Enable to publish event message that client unsubscribed a topic successfully.',
     zh: '是否开启客户端已成功取消订阅主题事件消息。',
   },
+  proxySubTip: {
+    zh: '通过代理订阅为客户端订阅主题时，不会经过 AuthZ 检查权限。',
+    en: 'The authorization will not check authority when subscribing topics for clients through proxy subscription.',
```
Comment (ysfscream) [inline src/i18n/Extension.js:314]: Authorization won't be checked when clients subscribe to topics through Auto Subscribe.

Comment (ysfscream) [reply src/i18n/Extension.js:314]: https://www.emqx.io/docs/en/v5.0/mqtt/mqtt-auto-subscription.html#configuration

Comment (ysfscream) [reply src/i18n/Extension.js:314]: https://www.emqx.io/docs/en/v5.0/mqtt/mqtt-auto-subscription.html#configuration

Comment (ysfscream) [reply src/i18n/Extension.js:314]: https://www.emqx.io/docs/en/v5.0/mqtt/mqtt-auto-subscription.html#configuration

Comment (Kinplemelon) [reply src/i18n/Extension.js:314]: Authorization is the one which checks permission, it doesn't need to be checked?

The authorization will not check permission when clients subscribe to topics through Auto Subscribe.
this?

Comment (ysfscream) [reply src/i18n/Extension.js:314]: > The authorization will not check permission when clients subscribe to topics through Auto Subscribe.

OK

**Pull request merged**